packages/components/src/color-picker/inputs.js

```diff
@@ -162,7 +162,7 @@ export class Inputs extends Component {
 			return (
 				<div className="components-color-picker__inputs-fields">
 					<Input
-						label={ __( 'Color value in hexadecimal' ) }
+						label={ __( 'Color value in Hex' ) }
```

Comment:
Isn't `hexadecimal` clearer to understand and easier to translate than `Hex` (which should be lowercase btw)?

Reply:
Industry standard is `Hex`, or in some case `HEX`. The full `hexadecimal` is usually used in documentation.

Reply:
Besides, We are already using `Hex` in other string, this PR is for creating consistency in color values.